[2025-03-29T14:56:34.274+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-29T14:56:34.440+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fist_dag_using_python_operator_v06.get_name scheduled__2025-01-11T00:00:00+00:00 [queued]>
[2025-03-29T14:56:34.518+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fist_dag_using_python_operator_v06.get_name scheduled__2025-01-11T00:00:00+00:00 [queued]>
[2025-03-29T14:56:34.537+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-03-29T14:56:35.042+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): get_name> on 2025-01-11 00:00:00+00:00
[2025-03-29T14:56:35.079+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fist_dag_using_python_operator_v06', 'get_name', 'scheduled__2025-01-11T00:00:00+00:00', '--job-id', '270', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmpcothkg74']
[2025-03-29T14:56:35.104+0000] {standard_task_runner.py:105} INFO - Job 270: Subtask get_name
[2025-03-29T14:56:35.091+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=662) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-29T14:56:35.171+0000] {standard_task_runner.py:72} INFO - Started process 681 to run task
[2025-03-29T14:56:35.622+0000] {task_command.py:467} INFO - Running <TaskInstance: fist_dag_using_python_operator_v06.get_name scheduled__2025-01-11T00:00:00+00:00 [running]> on host ba06481f48a9
[2025-03-29T14:56:36.248+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ARest' AIRFLOW_CTX_DAG_ID='fist_dag_using_python_operator_v06' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2025-01-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-11T00:00:00+00:00'
[2025-03-29T14:56:36.262+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-29T14:56:36.502+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-29T14:56:36.538+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-29T14:56:36.551+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fist_dag_using_python_operator_v06, task_id=get_name, run_id=scheduled__2025-01-11T00:00:00+00:00, execution_date=20250111T000000, start_date=20250329T145634, end_date=20250329T145636
[2025-03-29T14:56:36.799+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-29T14:56:36.918+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-29T14:56:36.928+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
